src/gui/qgsbrowserdockwidget.h

```diff
@@ -47,9 +47,10 @@ class GUI_EXPORT QgsBrowserDockWidget : public QgsDockWidget, private Ui::QgsBro
     /**
       * Constructor for QgsBrowserDockWidget
       * \param name name of the widget
+      * \param browserModel instance of the (shared) browser model
       * \param parent parent widget
       */
-    explicit QgsBrowserDockWidget( const QString &name, QWidget *parent SIP_TRANSFERTHIS = nullptr );
+    explicit QgsBrowserDockWidget( const QString &name, QgsBrowserModel *browserModel, QWidget *parent SIP_TRANSFERTHIS = nullptr );
```

Comment:
This is an API break if I'm not wrong, so I don't think this can be backported just like that. I guess this needs a new constructor introduced?

Reply:
@m-kuhn yes, you are right, thanks for pointing this out. I'll deal with than when backporting.